### PR TITLE
Update Rob's Bug Fixes - JK's RedBag's Solitude

### PR DESCRIPTION
**Pull request by robertgk2017**
Updated CRCs, due to update of my fixes mod.   Double checked the download from nexus!

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -26659,8 +26659,8 @@ plugins:
     clean:
       - crc: 0x57A26548
         util: 'SSEEdit v4.0.4b'
-      - crc: 0x930C1CE3
-        util: 'SSEEdit v4.0.4b'
+      - crc: 0x28033725
```
Comment (sibir-ine) [inline masterlist.yaml:26662]: The checksum of this plugin as downloaded is 0xBF58A6CE.

File: masterlist.yaml [hunk 2]
```diff
@@ -26659,8 +26659,8 @@ plugins:
...
+        util: 'SSEEdit v4.0.4c'
         
   - name: 'Dawn of RedBag''s Solitude'
```
Comment (sibir-ine) [inline masterlist.yaml]: I didn't catch this earlier, but, in #2614, you accidentally added some trailing whitespace. You can remove it here, or I can remove it in another commit, up to you.
```suggestion
        util: 'SSEEdit v4.0.4c'

  - name: 'Dawn of RedBag''s Solitude'
```

**Comment by robertgk2017** [comment]
Now Nexus seems to have caught up, download is now the correct file via correct CRC

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**